freeipa/src/test/java/com/sequenceiq/freeipa/util/KrbKeyTabGeneratorTest.java

```diff
@@ -0,0 +1,56 @@
+package com.sequenceiq.freeipa.util;
+
+import com.cloudera.thunderhead.service.usermanagement.UserManagementProto.ActorKerberosKey;
+import com.sequenceiq.cloudbreak.common.service.Clock;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
Are there any meaningful tests you want to bring from apache.directory?

Reply:
No longer needed.